src/components/NewCommentForm.tsx

```diff
@@ -75,25 +175,42 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames(
+              'textarea',
+              { 'is-danger': isCommentInputError },
+            )}
+            onChange={handleCommentChange}
```

Comment:
provide value for each form field. it will make form field controlled
```suggestion
            onChange={handleCommentChange}
            value={newComment.comment}
```
it will prevent such behavior. after submit form exists some unexpected behavior. in input exists some text, but in same time appears error
![chrome-capture-2023-10-26](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/33029864/eae9c111-bdcf-4f74-a9e8-c06a7192efea)